frontend/src/js/components/devices/dialogs/McuDeviceOnboarding.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { Typography } from '@mui/material';
+
+import { getOrganization } from '@northern.tech/store/selectors';
+import { setOnboardingApproach } from '@northern.tech/store/thunks';
+
+import CopyCode from '../../../common-ui/CopyCode';
```

Comment:
```suggestion
import CopyCode from '@northern.tech/common-ui/CopyCode';
```
hopefully we can soon get to make that switch to the actual package too...